R-package/R/lgb.cv.R

```diff
@@ -78,6 +81,7 @@ lgb.cv <- function(params = list()
                    , record = TRUE
                    , eval_freq = 1L
                    , showsd = TRUE
+                   , eval_train_metric = FALSE
```

Comment:
This is a break change from the perspective of anyone calling `lgb.cv()` with positional arguments. That's probably unlikely for a function with this many arguments, but I still would like to make this change in a non-breaking way.

Please move `eval_train_metric` to the end of the function signature, after `serializable`.

Reply:
Okay.